<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormSolvencia</class>
 <widget class="QWidget" name="FormSolvencia">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>450</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>650</width>
    <height>450</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>650</width>
    <height>450</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="lblCedula">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>10</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Baskerville Old Face</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Ingrese Cédula: </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditCedula">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>10</y>
     <width>191</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lblGrado">
   <property name="geometry">
    <rect>
     <x>95</x>
     <y>60</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Baskerville Old Face</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Grado: </string>
   </property>
  </widget>
  <widget class="QLabel" name="lblPeriodo">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>60</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Baskerville Old Face</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Año Escolar: </string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxGrado">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>60</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>1° Grado</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2° Grado</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>3° Grado</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>4° Grado</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>5° Grado</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>6° Grado</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>7° Grado</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>8° Grado</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>9° Grado</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="lineEditPeriodo">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>60</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSolvencia">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>210</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Imprimir</string>
   </property>
   <property name="icon">
    <iconset resource="recursos.qrc">
     <normaloff>:/images/impresora.png</normaloff>:/images/impresora.png</iconset>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="recursos.qrc"/>
 </resources>
 <connections/>
</ui>
